poscreators/integration-checklist.md

```diff
@@ -0,0 +1,22 @@
+
+| Country | Check | Description | Docs |
+| :----: | ----- | ----------- | --------- |
+| ![All markets](images/flag-eu.png) | **Sandbox Portal account created** | An account in the sandbox Portal was created to create test Middleware instances | [Getting Started Guide](README.md) |
+| ![All markets](images/flag-eu.png) | **API methods are implemented** | The POS System can call the Middleware's API via the protocol of choice | [Function structures](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/function-structures) & <br /> [Communication protocols](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/communication) |
+| ![All markets](images/flag-eu.png) | **Business sequences are implemented** | The required business cases (i.e. the `ftReceiptCases`, `ftChargeItemCases` and `ftPayItemCases`) are implemented. Depending on the POS system, not all cases might be required (only those which make sense for your use case). | [Receipt cases](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/reference-tables) & relevant country-specific appendices |
+| ![All markets](images/flag-eu.png) | **Receipt headers are included** | The correct values are passed for `ftCashBoxID`, `ftPosSystemID`, `cbTerminalID`, `cbReceiptReference` and `cbReceiptMoment`. | [Data structures](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/data-structures) & relevant country-specific appendices |
+| ![All markets](images/flag-eu.png) | **Closing receipts are implemented** | Daily, monthly and yearly closing receipts are implemented via the respective ´ftRecreiptCases`. | [Receipt cases](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/reference-tables) & relevant country-specific appendices |
+| ![All markets](images/flag-eu.png) | **Error handling is implemented** | The different types of errors (Middleware not reachable & signing device/service not reachable) are properly handled by the POS software | [Cash register integration](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/cash-register-integration) & relevant country-specific appendices |
+| ![All markets](images/flag-eu.png) | **Signatures are printed onto the physical receipts** | The `ftSignatureItems` returned by the Middleware are properly printed onto the physical receipts. Please note that generally all items need to printed, with some exceptions in Germany (more details available in the linked docs). | [Reference tables](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/reference-tables#type-of-signature-ftsignaturetype) & relevant country-specific appendices |
+| ![All markets](images/flag-eu.png) | **Production Portal account created** | An account in the production Portal was created to manage POS Systems. | [Getting Started Guide](README.md) |
```

Comment:
Should be done parallel or before SandBox.
1) Signing of PosCreator contract give some commitment
2) eMail address and account in D365 created for perfect support process

Reply:
Valid point, thanks! Moved it to the second step, as PosCreators might try to "non-bindingly" have a look before signing the contract, but it still be done before the integration :+1: